lib/routes/fruits.js

```diff
@@ -38,23 +38,32 @@ router.post('/fruits', validations.validateCreate, (request, response) => {
   });
 });
 
-router.put('/fruits/:id', validations.validateUpdate, (request, response) => {
-  const {name, stock} = request.body;
-  const {id} = request.params;
-  fruits.update({name, stock, id}).then((result) => {
+router.put('/fruits/:id', [parseRequestBodyToJson, validations.validateCreateUpdateRequest], (request, response) => {
+  const { name, stock } = request.body;
+  const { id } = request.params;
+  fruits.update({ name, stock, id }).then((result) => {
     if (result.rowCount === 0) {
       response.status(404);
       return response.send(`Unknown item ${id}`);
     }
-    return response.sendStatus(204);
```

Comment:
i don't think there is any reason to remove that `return` .  it technically doesn't need to be there, but the other methods have it, so we should keep it for consistency

Reply:
Ok.